testsuite/manualmode/src/test/resources/org/jboss/as/test/manualmode/management/persistence/testWithCli.xml

```diff
@@ -1,256 +1,256 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
It looks like these changes are related to the YAML support more than the new operation, @ehsavoie could you describe why they are not required?

Reply:
No, those changes are related to the fact that the cli was escaping " thus the XML was invalid. Replacing " by ' fixed that. It was of the initial issue we were trying to handle here.

Reply:
Thanks, it is clear to me now, I see it was intentional for the `read-config-as-xml` (deprecated one), and the new operation continues using `"`